internal/ignore/path_matcher.go

```diff
@@ -53,6 +53,7 @@ func CreateBuildContextFilter(m repoManifest) model.PathMatcher {
 type IgnorableManifest interface {
```

Comment:
hmmm... this seems redundant with `WatchableManifest`. Probably WM should implement IM

Reply:
Maybe? I kinda assumed they were added by the same person with some intent in mind, but I didn't spend time thinking about it since that seems orthogonal to this PR (though this PR slightly exacerbates any potential redundancy).